packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -81,22 +53,20 @@ export default function Classifier({
   }, [locale])
 
   useEffect(function onURLChange() {
-    const { workflows } = classifierStore
     if (workflowID) {
       console.log('starting new subject queue', { workflowID, subjectSetID, subjectID })
       workflows.setResources([workflowSnapshot])
-      workflows.selectWorkflow(workflowID, subjectSetID, subjectID, canPreviewWorkflows)
+      workflows.selectWorkflow(workflowID, subjectSetID, subjectID)
     }
-  }, [canPreviewWorkflows, subjectID, subjectSetID, workflowID])
```

Comment:
The `WorkflowStore` model needs updating, to either remove `canPreviewWorkflows` or set the default to `true`. Otherwise, previewing inactive workflows won't work.

Reply:
The tests that were removed from `Classifier` need to be moved up to `ClassifierContainer` too.